generate/src/contract/events.rs

```diff
@@ -190,17 +190,25 @@ fn expand_data_tuple(
 /// Expands into an `Events` type with method definitions for creating event
 /// streams for all non-anonymous contract events in the ABI.
 fn expand_filters(cx: &Context) -> Result<TokenStream> {
-    let filters = cx
+    let anonymous_events = cx
```

Comment:
Isn't the filter logic inverted? it looks like you are getting only events that are not anonymous.

Reply:
That was an oopsie with the name :stuck_out_tongue: